### PR TITLE
WIP: Adding possibility to cancel all children by resolve status 

### PR DESCRIPTION
**Pull request by grujawork**

### CODE REVIEW

File: contracts/SportMarkets/Rundown/TherundownConsumer.sol [hunk 1]
```diff
@@ -615,10 +627,15 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
         address _market,
         uint _outcome,
         uint8 _homeScore,
-        uint8 _awayScore
+        uint8 _awayScore,
+        uint8 _status
     ) internal {
         sportsManager.resolveMarket(_market, _outcome);
-        oddsObtainer.resolveChildMarkets(_market, _outcome, _homeScore, _awayScore);
+        if (_status == 201) {
```
Comment (danijelthales) [inline contracts/SportMarkets/Rundown/TherundownConsumer.sol:634]: Perhaps extract `201` into a variable that indicates what the status is for, e.g. "STATUS_RETIRED"

File: contracts/SportMarkets/Rundown/TherundownConsumer.sol [hunk 2]
```diff
@@ -615,10 +627,15 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
...
+            oddsObtainer.resolveChildMarkets(_market, 0, 0, 0);
```
Comment (danijelthales) [inline contracts/SportMarkets/Rundown/TherundownConsumer.sol:635]: I was considering to check if Totals can be resolved as over already based on the current result. e.g. if result sent is 2:1 and totals threshold was 3, perhaps we can resolve the totals markets as OVER.

Comment (danijelthales) [reply contracts/SportMarkets/Rundown/TherundownConsumer.sol:635]: But not sure tbh about this

File: contracts/SportMarkets/Rundown/TherundownConsumer.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (grujawork) [reply contracts/SportMarkets/Rundown/TherundownConsumer.sol:634]: i would leave it without sport id who knows when we will need that status

**Pull request merged**